addons/bestja_page_fixtures_ucw/templates/pages/become_volunteer.xml

```diff
@@ -117,19 +117,19 @@
                                 <div class="col-md-8 col-md-offset-2">
                                     <ul class="like_write">
                                         <li class="mb64">
-                                            <span>
```

Comment:
I think that there is no “like URL” on Facebook (only like plugin/button). Here you use share URL which can be generated if you know the URL of the page that you want to share. However maybe we should put here a direct URL to the Facebook fanpage?